src/dotnet/APIView/APIViewWeb/TagHelpers/RevisionTagHelper.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+using System;
+
+namespace APIViewWeb.TagHelpers
+{
+    [HtmlTargetElement(Attributes = "revision")]
+    public class RevisionTagHelper : TagHelper
```

Comment:
this is a generic tag helper to turn dates into readable strings used in multiple places, please don't specialize it for revisions.

Reply:
my bad